src/pages/Main/components/LastTransfers/LastNFTsTransfers.tsx

```diff
@@ -36,7 +40,19 @@ export const LastNFTsTransfers: VFC<LastTransfersProps> = ({
       searchString: prettifiedBlockSearchString,
     });
 
-  if (/[^$,-,.\d]/.test(searchString || '') || nftTransfersCount === 0) return null;
+  if (isNftTransfersFetching) {
+    return (
+      <SkeletonWrapper>
+        <Skeleton />
+      </SkeletonWrapper>
+    );
+  }
+
+  if (/[^$,-,.\d]/.test(searchString || '') || nftTransfersCount === 0) {
+    hideButton(false);
+    return <Stub />;
+  }
+  hideButton(true);
```

Comment:
Are you sure you should call hideButton from the root? 
It only can be called from useEffect or other callback func